src/vivarium_testing_utils/automated_validation/data_transformation/utils.py

```diff
@@ -40,3 +40,56 @@ def check_io(**model_dict: type) -> Callable[[F], F]:
 def series_to_dataframe(series: pd.Series[float]) -> pd.DataFrame:
     """Convert a Series to a DataFrame with the Series values as a single column."""
     return series.to_frame(name="value")
+
+
+def drop_extra_columns(raw_gbd: pd.DataFrame, data_key: str) -> pd.DataFrame:
+    """Format the output of a get_draws call to have expect index and value columns."""
+
+    value_cols = [col for col in raw_gbd.columns if "draw" in col]
+    # Population structure only has "population"
+    # Data should only have a "value" column when cached. Draws get converted when cached
+    if data_key == "population.structure":
+        # Population structure has only "population" column. Rename it
+        raw_gbd = raw_gbd.rename(columns={"population": "value"})
```

Comment:
@rmudambi I couldn't find a method that this renaming should belong to so I stuck it here but as discussed in the other comment, it needs to happen early on in data processing.